src/cbc_sdk/platform/devices.py

```diff
@@ -298,14 +317,25 @@ def nsx_remediation(self, tag, set_tag=True):
 
 
 class DeviceFacet(UnrefreshableModel):
-    """Represents a device field in a facet search."""
+    """
+    Represents a device field in a facet search.
+
+    *Faceting* is a search technique that categorizes search results according to common attributes. This allows
+    users to explore and discover information within a dataset, in this case, the set of devices.
+
+    Example:
+        >>> facets = api.select(Device).facets(['policy_id'])
+        >>> for value in facets[0].values_:
+        ...     count = len(value.query_devices())
```

Comment:
This seems excessive to make another query to get the count for each value as it has the total count in the response which should be stored in DeviceFacetValue as `total`

https://developer.carbonblack.com/reference/carbon-black-cloud/platform/latest/devices-api/#facet-devices

Reply:
That worked, so I'm doing it that way.